direct_test.go

```diff
@@ -167,15 +167,15 @@ func TestHostAliasesBasic(t *testing.T) {
 	}{
 		{
 			"http://fff.cloudsack.biz/foo",
-			"Get http://fff.cloudsack.biz/foo: No alias for host fff.cloudsack.biz",
+			"Get \"http://fff.cloudsack.biz/foo\": No alias for host fff.cloudsack.biz",
```

Comment:
If you want to avoid having to escape nested quotes in go, you can use backtick quotes like this:

```go
`Get "http://fff.cloudsack.biz/foo": No alias for host fff.cloudsack.biz`
```

Reply:
Yay that's much cleaner - thanks!